.env.example

```
@@ -26,9 +26,11 @@ EMAIL_SMTP_PASSWORD=
 
 JWT_ISSUER=SBay
 JWT_AUDIENCE=SBayClients
+# JWT_SECRET signs access JWTs only. Refresh tokens are opaque random strings
+# stored hashed in Postgres; JWT_REFRESH_TOKEN_DAYS controls their lifetime.
 JWT_SECRET=dev_only_change_me_32_bytes_minimum_value
 JWT_EXP_MINUTES=60
-JWT_REFRESH_TOKEN_DAYS=30
+JWT_REFRESH_TOKEN_DAYS=180
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_REFRESH_TOKEN_DAYS key should go before the JWT_SECRET key

(UnorderedKey)